src/main/java/frc/robot/utils/debugging/WheelRadiusCharacterization.java

```diff
@@ -1,108 +1,113 @@
-//   private double accumGyroYawRads = 0.0;
-
-//   private double[] startWheelPositions;
-
-//   private double currentEffectiveWheelRadius = 0.0;
-
-//   private DoubleSupplier gyroYawRadsSupplier;
-
-//   public WheelRadiusCharacterization(Drive drive, Direction omegaDirection) {
-//     this.drive = drive;
-//     this.omegaDirection = omegaDirection;
-//     gyroYawRadsSupplier = () -> drive.getRotation().getRadians();
-//     addRequirements(drive);
-//   }
-
-//   @Override
-//   public void initialize() {
-//     // Reset
-//     lastGyroYawRads = gyroYawRadsSupplier.getAsDouble();
-//     accumGyroYawRads = 0.0;
-
-//     startWheelPositions = getWheelPositions();
-
-//     omegaLimiter.reset(0);
-//   }
-
-//   @Override
-//   public void execute() {
-//     // Run drive at velocity
-//     drive.runSwerve(
-//         new ChassisSpeeds(
-//             0.0,
-//             0.0,
-//             omegaDirection.value * (characterizationSpeed.get() * Drive.MAX_ANGULAR_SPEED_MPS)));
-
-//     // Get yaw and wheel positions
-//     accumGyroYawRads += MathUtil.angleModulus(gyroYawRadsSupplier.getAsDouble() -
-// lastGyroYawRads);
-//     lastGyroYawRads = gyroYawRadsSupplier.getAsDouble();
-//     double averageWheelPosition = 0.0;
-//     double[] wheelPositiions = getWheelPositions();
-//     for (int i = 0; i < 4; i++) {
-//       averageWheelPosition += Math.abs(wheelPositiions[i] - startWheelPositions[i]);
-//     }
-//     averageWheelPosition /= 4.0;
-
-//     currentEffectiveWheelRadius = (accumGyroYawRads * driveRadius) / averageWheelPosition;
-//     Logger.recordOutput("Drive/RadiusCharacterization/DrivePosition", averageWheelPosition);
-//     Logger.recordOutput("Drive/RadiusCharacterization/AccumGyroYawRads", accumGyroYawRads);
-//     Logger.recordOutput(
-//         "Drive/RadiusCharacterization/CurrentWheelRadiusInches",
-//         Units.metersToInches(currentEffectiveWheelRadius));
-//   }
-
-//   @Override
-//   public void end(boolean interrupted) {
-//     if (accumGyroYawRads <= Math.PI * 2.0) {
-//       System.out.println("Not enough data for characterization");
-//     } else {
-//       System.out.println(
-//           "Effective Wheel Radius: "
-//               + Units.metersToInches(currentEffectiveWheelRadius)
-//               + " inches");
-//     }
-//   }
-
-//   public double[] getWheelPositions() {
-//     SwerveModulePosition[] positions = drive.getModulePositions();
-//     double[] doublePosition = new double[4];
-
-//     for (int i = 0; i < 4; i++) {
-//       startWheelPositions[i] = positions[i].angle.getRadians();
-//     }
-
-//     return doublePosition;
-//   }
-// }
+import java.util.function.DoubleSupplier;
+import lombok.RequiredArgsConstructor;
+import org.littletonrobotics.junction.Logger;
+
+
+// UN COMMENT DRIVEBASE RELATED COMMENTS, SUCH AS ON LINES 29, 64-70 AND THE getWheelPositions() FUNCTION
+public class WheelRadiusCharacterization extends Command {
```

Comment:
Is this file only used for drive base?  It should probably be in the drive base template then.

Reply:
It would overcrowd the drive folder, and it's a one-time test, so I didn't see the value in leaving it in the drive folder. That is just what my brain told me at the time, that's up for change based on any standard we decide for utilities.